RetailCoder.VBE/Inspections/QuickFixes/QuickFixHelper.cs

```diff
@@ -10,28 +10,32 @@
 
 namespace Rubberduck.Inspections.QuickFixes
 {
-    class AssignedByValParameterQuickFixHelper
+    class QuickFixHelper
```

Comment:
I think, because its tightly coupled with the `FoobarQuickFix` implementations - and IMO that's a fine coupling to have here (although a few tests for it would be golden), so we should make the class `static` and remove all state: the `target` and `selection` become parameters of the members that need them, and the members become `static` as well, and then some could also be implemented as extension methods that extend `ICodePane`, for example.